app/move/controllers/create/save.js

```diff
@@ -23,14 +23,21 @@ class SaveController extends CreateBaseController {
         'errors',
         'errorValues',
       ])
-      const move = await moveService.create(data)
+
+      const profile = await profileService.create(data.person.id, {
+        assessment_answers: data.assessment,
+      })
+      delete data.person
+      delete data.assessment
```

Comment:
Isn't preferable to copy first and alter later?

Reply:
Could also use `_.omit()` to remove the keys you don't want to send to `moveService.create()`?

Reply:
Yes, good point. I was being lazy. I'll rejig it